bazel/kotlin_test.bzl

```diff
@@ -6,6 +6,7 @@ load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kt_jvm_test")
 # 1. Avoiding the need to declare the test_class which requires a fully qualified class name (example below)
 # 2. Avoiding the need to redeclare common unit testing dependencies like JUnit
 # 3. Ability to run more than one test file per target
+# 4. Ability to test internal entities which are envoy mobile dependencies
```

Comment:
This doesn't quite make sense

Reply:
`Ability to test internal envoy mobile entities`?

Reply:
yea that would work